geotags/views.py

```diff
@@ -47,16 +49,18 @@
             ORDER BY sounds_sound.created DESC
                      %(end)s"""
 
-def generate_json(sound_queryset):
-    # When using Django Sound queries:
-    #sounds_data = [[s.id, s.geotag.lat, s.geotag.lon] for s in sound_queryset]
+def generate_bytearray(sound_queryset):
+    # sounds as bytearray
+    packed_sounds = cStringIO.StringIO()
+    for sound in sound_queryset:
+        packed_sounds.write(struct.pack("i", sound.id))
+        packed_sounds.write(struct.pack("i", int(sound.lat*1000)))
```

Comment:
The raw query gets id, lat and lon in SELECT, thus it should work in most views, but `geotags_box_json ` https://github.com/MTG/freesound/blob/master/geotags/views.py#L75 does not make a raw query so it will fail. I guess this view should be refactored to use the raw query so it works like the others. Otherwise it should be annotated somehow so s.lat and s.lon are available.